zenodo/modules/pages/templates/zenodo_pages/contact_form.html

```diff
@@ -80,6 +80,8 @@
 {%- endblock page_body %}
 
 {%- block javascript %}
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.0/jquery.min.js"></script>
```

Comment:
You can skip this, since jQuery is by default loaded from the `zenodo.js` bundle (included through [require.js](https://github.com/zenodo/zenodo/blob/master/zenodo/modules/theme/bundles.py#L62-L63) from [this file](https://github.com/zenodo/zenodo/blob/master/zenodo/modules/theme/static/js/zenodo.js#L23))